File: 1]. DSA/1]. Data Structures/01]. Array/JavaScript/_005)_Left_Rotate_Array_by_1.js
function lRotateOne(arr, n) {
    let temp = arr[0];
    for (let i = 1; i < n; i++) {
        arr[i - 1] = arr[i];
    }
    arr[n - 1] = temp;
}

let arr = [1, 2, 3, 4, 5];
let n = 5;
console.log(`Before Left Rotation -> ${arr}`);
lRotateOne(arr, n);
console.log(`After '1' Left Rotation -> ${arr}`);

File: 1]. DSA/1]. Data Structures/01]. Array/Python/_005)_Left_Rotate_Array_by_1.py
"""
Function to rotate an the first element
"""


def left_rotate(arr, num):
    tmp = arr[0]

    for index in range(1, num):
        arr[index - 1] = arr[index]

    arr[num - 1] = tmp


arr = [1, 2, 3, 4, 5]
num = 2

print("Before Left Rotation:", arr)

left_rotate(arr, num)
print("After '1' Left Rotation:", arr)
